dev/tests/js/framework/JsTestRunner.php

```diff
@@ -0,0 +1,537 @@
+<?php
+
+class JsTestRunner
```

Comment:
Moving scripting behavior to class doesn't help. Also because all of the helper methods are private and on the same class, it's impossible to configure it differntly. Also different entry points should get different instances of different classes. writeJsTestRunnerConfig() has nothing to do with runTests()